Resources/DesktopModules/ResourceManager/App_LocalResources/settings.ascx.nl-NL.resx

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ControlTitle_settings.Text" xml:space="preserve" lastModified="2021-09-01 11:38:58">
+    <value>ResourceManager instellingen</value>
+  </data>
+  <data name="lblHomeFolder.Help" xml:space="preserve" lastModified="2021-09-01 11:38:58">
+    <value>Selecteer de basismap voor de module</value>
+  </data>
+  <data name="lblHomeFolder.Text" xml:space="preserve" lastModified="2021-09-01 11:38:58">
+    <value>Home map</value>
+  </data>
+  <data name="lblMode.Help" xml:space="preserve" lastModified="2021-09-01 11:38:58">
+    <value>De modus van de module is: normaal, groep of gebruiker. De groepsmodus associeert de module met de groepsmap, de gebruikersmodus associeert  de module met de gebruikersmap.</value>
```

Comment:
e modus van de module is: normaal, groep of gebruiker. De groepsmodus associeert de module met de groepsmap, de gebruikersmodus associeert de module met de gebruikersmap.

(spatie teveel)